media-sound/supercollider/supercollider-3.11.0.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake readme.gentoo-r1 xdg-utils
+
+DESCRIPTION="An environment and a programming language for real time audio synthesis."
+HOMEPAGE="https://supercollider.github.io/"
+SRC_URI="https://github.com/supercollider/supercollider/releases/download/Version-${PV}/SuperCollider-${PV}-Source.tar.bz2"
+
+LICENSE="GPL-2 gpl3? ( GPL-3 )"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="cpu_flags_x86_sse cpu_flags_x86_sse2 debug emacs +fftw gedit +gpl3 jack qt5 server +sndfile static-libs vim X zeroconf"
+
+REQUIRED_USE="qt5? ( X )"
```

Comment:
is that still accurate? quite often this is accompanied by a dependency on qtx11extras but at least from the ebuild that does not seem to be the case.

Reply:
Upstream definitely has X as a dependency of the Qt GUI in their buildsystem (for Linux), and a pile of codepaths referencing Q_WS_X11 there (and not a trace of wayland). There is also optional X support outside of the GUI.

I'm not confident we can do much about this here.